packages/browser-repl/src/components/shell.tsx

```diff
@@ -262,6 +264,7 @@ const _Shell: ForwardRefRenderFunction<EditorRef | null, ShellProps> = (
           ...(outputRef.current ?? []),
           ...result.map(
             (entry): ShellOutputEntry => ({
+              key: lastKey++,
```

Comment:
Also needs to be accounted for on L362/L396, right? Maybe these are easier to catch if we don't make this an optional property?

Reply:
yeah I plan on making it required - that was just temporary so I didn't have to change who knows how many places to get things to build and test it out.

Reply:
Actually I think I overreacted. I saw a bunch of tabs in vscode go red and thought it would touch way more places, but it was just a few.